src/components/custom/_signup-affiliate.js

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Flex } from '../containers'
+import AgreementLabel from './_agreement-label'
+import { Input, Button } from 'components/form'
+import { Header, LinkText, LocalizedLinkText } from 'components/elements'
+import { localize } from 'components/localization'
+import device from 'themes/device.js'
+
+const StyledFlex = styled(Flex)`
+    height: 450px;
+    background-color: var(--color-white);
+    border-radius: 6px;
+    box-shadow: 0 1.6rem 2rem 0 rgba(0, 0, 0, 0.1);
+
+    @media ${device.tabletL} {
+        margin: 0 auto;
+        padding: 40px 16px 0;
+        width: 360px;
+        min-width: 360px;
+        height: 392px;
+    }
+`
+const StyledNoteFlex = styled(Flex)`
+    min-width: 328px;
+    height: 36px;
+    border-radius: 4px;
+    background-color: rgba(248, 248, 249, 1);
+
+    @media ${device.mobileL} {
+        height: 34px;
+        padding: 8px 26px;
+    }
+`
```

Comment:
Generally we shouldn't set fixed heights for these containers/wrappers as we need to support multiple languages that may have longer texts in others. Generally, a rule of thumb is to set a min-height of these hard-coded heights and allow height to expand dynamically with `fit-content`/`auto`.

Reply:
Refactored forms wrappers height / width in that way.